pkgs/development/python-modules/apricot-select/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, scipy
+, tqdm
+, scikit-learn
+, nose
+, numba
+, torch
+, python
+}:
+
+
+buildPythonPackage rec {
+  pname = "apricot-select";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-O/hy1D7pavFByeTEDkNZqmyl4wIq5DsqiqRrJJR7i9g=";
+  };
+
+  propagatedBuildInputs = [ nose numpy scipy tqdm numba torch scikit-learn ];
+
+  pythonImportsCheck = [ "apricot" ];
+
+  nativeCheckInputs = [ python ];
+
+  checkPhase = ''
+    ${python.interpreter} setup.py test
+  '';
```

Comment:
Can you check whether `unittestCheckHook` or `pytestCheckHook` work instead? Caling `setup.py` with `test`is deprecated.

Reply:
`unittestCheckHook` works for this.